terraform/modules/front_door/waf.tf

```diff
@@ -152,4 +152,26 @@ resource "aws_wafv2_web_acl" "this" {
       sampled_requests_enabled   = true
     }
   }
+
+  rule {
+    name     = "rate-limiting"
+    priority = 7
+
+    action {
+      block {}
+    }
+
+    statement {
+      rate_based_statement {
+        aggregate_key_type = "IP"
+        limit              = 1000

```

Comment:
I'm not sure about a limit yet, but wanted to get the implementation reviewed while a determine a sensible threshold

Reply:
We may need to increase this limit during performance testing temporarily, I've added a note about this on the ticket [here](https://dluhcdigital.atlassian.net/browse/CLDC-2737). Hopefully not, but I recall on EPB we hit the limit because all our requests were being sent from a single IP during testing.

Reply:
I think I was expecting this to only apply to the login path (actually that should probably be all paths you can reach without being logged in, so e.g. reset password as well) - which does most of the protection (assuming relative trust in our users), and allows us to set a much lower limit (because number of requests for legitimate use will be lower). 

Delta has an example of that that I was working from - they actually have an overall rate limit and a lower limit for login routes. 

I appreciate the ticket isn't particularly well laid out, and an overall limit is separately a good thing - in hindsight I'm actually fairly tempted to have both if you're up for implementing that as well?

Reply:
Sorry - looks like I hadn't read the ticket properly! But yes up for implementing all of that 👍